proxy.config.json

```diff
@@ -1,4 +1,14 @@
 {
+  "/api/markups/*": {
```

Comment:
Do we really need this? The ```"/api/*"``` below should handle all/any routes going through api calls